tck/src/main/java/ee/jakarta/tck/data/framework/signature/DataSignatureTestRunner.java

```diff
@@ -229,8 +229,11 @@ public void signatureTest() throws Fault {
 
 //            mapFileAsProps = getSigTestDriver().loadMapFile(mapFile);
 
+            String javaVersion = System.getProperty("java.specification.version");
+            String sigRsrc = SIG_RESOURCE_PACKAGE.replace(".", "/") + "/" + SIG_FILE_NAME;
+            sigRsrc += "_" + javaVersion;
```

Comment:
So I think this will break the signature tests for core/web/full profile.
https://github.com/jakartaee/data/blob/352d6366a6b071d521c244fb75a34670ce3f2fba/tck/src/main/java/ee/jakarta/tck/data/framework/arquillian/extensions/TCKArchiveProcessor.java#L109

When we deploy the app to the container to run the signature test we rename the resource without `_17` and `_21`

Reply:
Ok, I added a fallback to the base SIG_FILE_NAME if the jvm specification version cannot be found.